.github/workflows/jacoco.yml

```diff
@@ -0,0 +1,51 @@
+name: Coverage
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
```

Comment:
we do not need to set a name to checkout@v4

Reply:
Already remove the name.Only left `      - uses: actions/checkout@v4
        with:
          persist-credentials: false`